README.md

```diff
@@ -16,35 +16,55 @@
 * [3rd Party Libraries](docs/3rd-party-libs.md)
 
 ## Project Description
-The Bodacious Advisor for Cuisine Over N-joyment ranks the restaurants within London based on Twitter reviews.
+
+Online reviews have risen as one of the major factors that influence purchase behaviours of today’s consumers. Thanks to sites like Yelp and, as well as social networks like Twitter. Those reviews provide strong word-of-mouth power that can draw or drive away potential customers.
```

Comment:
Reviews on specialized websites like Yelp and social networks such as  Twitter, provide strong word-of-mouth power that...